packages/fxa-settings/src/styles/tailwind.css

```diff
@@ -42,7 +43,7 @@ progress::-webkit-progress-inner-element {
 }
 
 progress::-webkit-progress-value {
-   border-radius: 10px;
+  border-radius: 10px;
```

Comment:
Awh man, I guess this snuck in with a previous PR (this shouldn't be a px value). Unrelated to your changes though.

Reply:
Do you want me to change it something else while it is on our radar?